src/reduce/problem_description.cpp

```diff
@@ -35,20 +35,23 @@ namespace reduce {
 
 NetworkConfig ProblemDescription::MakeNetworkConfig() const
 {
-    auto xlength = xDesc.GetLengths();
-    auto ylength = yDesc.GetLengths();
+    auto xlength      = xDesc.GetLengths();
+    auto outputlength = reduceDesc.GetLengths();
 
     auto reduce_size  = xlength[dim];
-    auto output_numel = std::accumulate(
-        ylength.begin(), ylength.end(), static_cast<size_t>(1), std::multiplies<size_t>());
-    auto dtype = xDesc.GetType();
+    auto output_numel = std::accumulate(outputlength.begin(),
+                                        outputlength.end(),
+                                        static_cast<size_t>(1),
+                                        std::multiplies<size_t>());
+    auto dtype        = xDesc.GetType();
 
     std::ostringstream ss;
 
     ss << "dtype" << dtype;
     ss << "dim" << dim;
     ss << "reduce_size" << reduce_size;
     ss << "output_numel" << output_numel;
+    ss << "reduceExtremeOp" << reduceExtremeOp;
```

Comment:
[Recommendation] In the context of reduce operations (i.e. like here, in the problem description of the reduce operation), "reduce" or "reduceExtreme" is superfluous.
```suggestion
    ss << "size" << size;
    ss << "output_numel" << output_numel;
    ss << "op" << op;
```
Everywhere. Just a recommendation; up to you.

Reply:
fixed it.

Reply:
[resolved]